redis_cluster.c

```diff
@@ -2034,17 +2034,16 @@ PHP_METHOD(RedisCluster, _masters) {
         if (node == NULL) break;
 
         zval z, *z_sub = &z;
-#if (PHP_MAJOR_VERSION < 7)
-        MAKE_STD_ZVAL(z_sub);
-#endif
+
+        REDIS_MAKE_STD_ZVAL(z_sub);
         array_init(z_sub);
 
         add_next_index_stringl(z_sub, ZSTR_VAL(node->sock->host), ZSTR_LEN(node->sock->host));
         add_next_index_long(z_sub, node->sock->port);
         add_next_index_zval(z_ret, z_sub);
     } ZEND_HASH_FOREACH_END();
 
-    RETVAL_ZVAL(z_ret, 1, 0);
+    RETVAL_ZVAL(z_ret, 0, 0);
```

Comment:
I think in this method we can use `return_value` directly and remove `z_ret` at all. No?

Reply:
Should be able to, yes.  `_masters` is easy since it doesn't have to worry about `multi` weirdness.  :smile: